lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -13,22 +13,22 @@ class CategoryRenderer {
    * @param {!DetailsRenderer} detailsRenderer
    */
   constructor(dom, detailsRenderer) {
-    /** @private {!DOM} */
-    this._dom = dom;
-    /** @private {!DetailsRenderer} */
-    this._detailsRenderer = detailsRenderer;
-    /** @private {!Document|!Element} */
-    this._templateContext = this._dom.document();
-
-    this._detailsRenderer.setTemplateContext(this._templateContext);
+    /** @protected {!DOM} */
+    this.dom = dom;
```

Comment:
`yarn compile-devtools` forces me to remove `_` prefix from protected variables 😿